query-service-impl/src/test/resources/trino_request_handler.conf

```diff
@@ -0,0 +1,34 @@
+{
+  name = span-event-view-handler
```

Comment:
```suggestion
  name = span-event-view-trino-handler
```

Reply:
In case If we are merging the base `application.conf` + this one.

Reply:
No merging. This conf is used for testing standalone class `QueryRequestToTrinoSQLConverter`

Reply:
Renamed handler, though I couldn't see any enforcement of handler name to be unique.